image_to_json.py

```diff
@@ -1,23 +1,65 @@
 #!/usr/bin/env python
-"""
-Opens image and converts to csv data using the PIL library
+"""Converts image to json-like file.
+
+Usage:
+    image_to_json.py (-i FILE) [-o FILE] [-s K] [-f FILTER]
+
+Options:
+    -h --help                   Shows this screen.
+    -i FILE --input=FILE        Input image file.
+    -o FILE --output=FILE       Output json file. [default: image.json]
+    -s K --size=K               Resize image in K. [default: False]
+    -f FILTER --filter=FILTER   Resampling filter, NEAREST, BILINEAR, BICUBIC or LANCZOS. [default: NEAREST]
 """
 
-import Image
-from sys import argv, exit
+
+import os
+import sys
+import getopt
+from docopt import docopt
```

Comment:
Some of those imports were left behind from a different approach I had... Pretty lousy!